src/homepage/howTo/mail-processing.html

```diff
@@ -0,0 +1,287 @@
+---
```

Comment:
maybe I missed something but why do we have static html with all links and mnu duplicated ? isn't what jekyll is supposed to do for us ?